### DISCUSSION TITLE
tutorials/ui/size_and_anchors

### DISCUSSION THREAD

**Discussion by giscus** [godotengine/godot-docs-user-notes#172] [User-contributed notes]
# tutorials/ui/size_and_anchors

If a game was always going to be run on the same device and at the same resolution, positioning controls would be a simple matter of setting the position and size of each one of them. Unfortunately...

https://docs.godotengine.org/en/4.3/tutorials/ui/size_and_anchors.html

**Comment by fc470505146** [comment]
Margin was renamed to AnchorOffsets I guess

**Reply by tot121212** [reply]
Yes, apparently so.

**Comment by Ena-ki** [comment]
I feel like this assumes that the reader already knows what an anchor / margin is, so it was kinda confusing.

**Comment by shegedep** [comment]
Ohh, the Anchor is a BOX, not a point.
I spent ages here trying to parse what I was looking at/reading.

**Reply by TrueMatrixPower** [reply]
You are Absolutely correct, it's a BOX, like Box Model in HTML, nothing speical.

**Comment by liyiz** [comment]
I hope this helps someone in the future, the gdscript example under "Centering a control" mentions `load("res://icon.png")` but Godot 4.6 now has its icon as an svg, so just make sure to amend it to `load("res://icon.svg")`!

**Reply by Calinou** [reply]
https://github.com/godotengine/godot-docs/pull/11936 recently fixed this in the docs. It might not appear on `stable` yet though (until 4.7 is released).
